can-auth/vatican.c

```diff
@@ -76,20 +103,20 @@ int VULCAN_FUNC vatican_mac_create(uint8_t *mac, uint16_t id, uint8_t* msg,
     ad.doubles[1] = id;
     for (i=0; i < CAN_PAYLOAD_SIZE; i++)
         ad.bytes[VATICAN_NONCE_SIZE+CAN_SID_SIZE+i] = (i < len) ? msg[i] : 0x00;
-    pr_debug_buf(ad.bytes, VATICAN_AD_SIZE, INFO_STR("AD"));
- 
+    //pr_debug_buf(ad.bytes, VATICAN_AD_SIZE, INFO_STR("AD"));
```

Comment:
use TRAVIS if you want to reduce debug output

https://github.com/sancus-pma/vulcan/blob/master/can-auth/vulcan.h#L46